apps/admin/backend/src/app.cdf_cvr_files.test.ts

```diff
@@ -7,15 +7,20 @@ import { basename, join } from 'path';
 import * as fs from 'fs';
 import { CVR_BALLOT_LAYOUTS_SUBDIRECTORY } from '@votingworks/backend';
 import { vxBallotType } from '@votingworks/types/src/cdf/cast-vote-records';
+import {
+  castVoteRecordHasWriteIns,
+  castVoteRecordVoteIsWriteIn,
+} from '@votingworks/utils';
 import {
   buildTestEnvironment,
   configureMachine,
   mockCastVoteRecordFileTree,
   mockElectionManagerAuth,
 } from '../test/app';
 import { modifyCastVoteRecordReport } from '../test/utils';
+import { getWriteInsFromCastVoteRecord } from './util/cvrs';
 
-jest.setTimeout(30_000);
+jest.setTimeout(60_000);
```

Comment:
This was a flaky case - it would sometimes hit above 30 seconds and sometimes not. It is usually not going close to 60 seconds. Still super slow though!

Reply:
Gotcha, that makes sense. Do we need to have 3000 records in the fixture? Or is there a more minimal one we could use?

Reply:
We don't need a fixture with 3000 records in _this_ test, but I think we do need a big fixture once we start doing tallying on the backend. That way we can have a report with the full mix of context options (whatever we decide that full mix to be) and then some. For now, I lean toward keeping it as is since it will be compatible with tally testing.

I just ran the tests locally and the times don't look too bad. It's only the big integration test that takes a really long time, and I think that's OK.

```  
  ✓ happy path - mock election flow (29026 ms)
  ✓ adding a duplicate file returns OK to client but logs an error (10497 ms)
  ✓ handles file with previously added entries by adding only the new entries (2105 ms)
  ✓ error if path to report is not valid (15 ms)
  ✓ error if report has invalid directory structure (26 ms)
  ✓ error if report metadata is not parseable (7869 ms)
  ✓ error if adding test report while in official mode (2035 ms)
  ✓ error if adding official report while in test mode (10138 ms)
  ✓ error if a cast vote record not parseable (1033 ms)
  ✓ error if a cast vote record is somehow invalid (1007 ms)
  ✓ error if cast vote records from different files share same ballot id but have different data (11039 ms)
  ✓ error if a layout is invalid (1086 ms)
```

Reply:
I actually had noticed the slow happy path test when I was working on something that touched it. That's more why I called it out now. I do think 30s is on the long side for a test case, but it's not a blocking concern